cmake/osinfo.cmake

```diff
@@ -0,0 +1,12 @@
+# Get the operating system information. Generally, CMake does a good job of
+# this. Sometimes, though, it doesn't give enough information. These steps will
+# distinguish between the UNIX variants. Otherwise, use the CMake variables
+# such as WIN32 and APPLE and CYGWIN.
+# Sets OS_IS_64BIT if the operating system is 64-bit.
+# Sets LINUX if the operating system is Linux.
+string(REGEX MATCH "Linux" OS_IS_LINUX ${CMAKE_SYSTEM_NAME})
```

Comment:
I just searched and can't find this stuff being used anywhere.  Perhaps it can all be ditched.

Reply:
I think it is old junk from early CMake 2.x days.

Reply:
Interesting. I wasn't sure of the origin. Just this particular line @abellgithub or the whole thing?

Reply:
Forget it. I see it's really just checking that one thing.

Reply:
I meant that entire block of code that you were moving.